### ISSUE TITLE
Support img tag

### ISSUE THREAD

**Issue by 0x6b** [0x6b/copy-selection-as-markdown#1]

**Comment by 0x6b** [comment]
Ref: [webextensions-examples/context-menu-copy-link-with-types at master · mdn/webextensions-examples](https://github.com/mdn/webextensions-examples/tree/master/context-menu-copy-link-with-types)

**Comment by allenyllee** [comment]
Can this be help?
https://github.com/euangoddard/clipboard2markdown/tree/master

I use it to convert my copied content, including image, and it convert img tag correctly.

**Comment by allenyllee** [comment]
It seems that there is an another addon can do that: [BlackGlory/Copycat: Copy content from web powerful than ever before.](https://github.com/BlackGlory/Copycat)

**Comment by 0x6b** [comment]
Sorry for delay on my end as I have some health issue recently. Anyway thanks a lot for your comments @allenyllee. I just updated with preliminary img tag support and tagged it as [v0.2.0](https://github.com/0x6b/copy-selection-as-markdown/releases/tag/v0.2.0). https://addons.mozilla.org/en-US/firefox/addon/copy-selection-as-markdown/ is just updated as well. Please have a look into the new one and let me know if you have any issue. Thanks!

**Comment by allenyllee** [comment]
@0x6b  That's great! In some cases, your addon even more correct then copycat. 
For example, I want to copy [this page](https://stats.stackexchange.com/questions/235528/backpropagation-with-softmax-cross-entropy) which contains latex formula:

![](https://screenshotscdn.firefoxusercontent.com/images/2b1748f4-40cf-4903-a3d5-a4e2a2daf59c.png)

I can get the origin latex formula via your addon:
> ```
> I'm trying to understand how backpropagation works for a softmax/cross-entropy output layer.
> 
> The cross entropy error function is
> 
> E(t,o)=−∑jtjlogoj
> 
> E(t,o)=-\\sum\_j t\_j \\log o_j
> ```

instead of the html element via copycat:
> ```
> I'm trying to understand how backpropagation works for a softmax/cross-entropy output layer.
> 
> The cross entropy error function is
> 
> <nobr aria-hidden="true">E(t,o)=−∑jtjlogoj</nobr>
> ```

But I think it can be improved by adding $$..$$ or $...$ and replace double slash with single slash and remove the plain text, like this:

> ```
> I'm trying to understand how backpropagation works for a softmax/cross-entropy output layer.
> 
> The cross entropy error function is
> 
> $$E(t,o)=-\sum_j t_j \log o_j$$
> ```

Maybe I should open a new issue for this requirement.

**Comment by 0x6b** [comment]
It's interesting use case. Let me see how it can be implemented. I'll open a new issue for you later.
